examples/ipam-vpc/README.md

```diff
@@ -0,0 +1,171 @@
+# VPC with IPAM pool
+
+Configuration in this directory creates set of VPC resources using the CIDR provided by an IPAM pool.
+
+Note: Due to the nature of vending CIDR blocks from an IPAM pool, the IPAM pool must exist prior to creating a VPC using one of the CIDRs from the pool.
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply -target=aws_vpc_ipam_preview_next_cidr.this # CIDR pool must exist before assigning CIDR from pool
+$ terraform apply
+```
+
+To destroy this example you can execute:
+
+```bash
+$ terraform destroy -target=module.vpc # destroy VPC that uses the IPAM pool CIDR first
+$ terraform destroy
```

Comment:
not sure why, but deleting the `aws_vpc_ipam_pool_cidr.this` takes forever 🤷🏽‍♂️

Reply:
It would be great to combine these two `terraform destroy` into one, as we have in all other modules, but I could not find a way to do this now. Let's leave it as it is now.

Reply:
> not sure why, but deleting the `aws_vpc_ipam_pool_cidr.this` takes forever 🤷🏽‍♂️

The service holds the allocation record for up to 30m after a vpc releases it. It's possible to ignore the allocation and delete faster but conflicts with terraforms paradigm

Reply:
Oh, good to know!